src/main/java/no/nav/pto/veilarbportefolje/elastic/ElasticQueryBuilder.java

```diff
@@ -83,13 +85,18 @@ static void leggTilManuelleFilter(BoolQueryBuilder queryBuilder, Filtervalg filt
         if (filtervalg.harCvFilter()) {
             if (filtervalg.cvJobbprofil.equals(CVjobbprofil.HAR_DELT_CV)) {
                 MatchQueryBuilder harDeltCV = matchQuery("har_delt_cv", true);
-                MatchQueryBuilder cvEksistere = matchQuery("cv_eksistere", true);
-                queryBuilder.must(harDeltCV).must(cvEksistere);
+                queryBuilder.must(harDeltCV);
+                if (FeatureToggle.erCvEksistereIProd(unleashService)) {
+                    MatchQueryBuilder cvEksistere = matchQuery("cv_eksistere", true);
+                    queryBuilder.must(cvEksistere);
+                }
             } else {
                 BoolQueryBuilder orQuery = QueryBuilders.boolQuery();
                 orQuery.should(matchQuery("har_delt_cv", false));
-                orQuery.should(matchQuery("cv_eksistere", false));
-                orQuery.minimumShouldMatch(1);
+                if (FeatureToggle.erCvEksistereIProd(unleashService)) {
+                    orQuery.should(matchQuery("cv_eksistere", false));
+                    orQuery.minimumShouldMatch(1);
```

Comment:
was this necessary?

Reply:
minimalShouldMatch? Probably now...I will remove it